contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -173,6 +172,47 @@ In addition, `jws-kubeconfig-<token-id>-hash` will be set to the MD5 hash of the
 
 This controller will also delete tokens that are past their expiration time.
 
+## `kubeadm` UX
+
+We extend kubeadm with a set of flags and helper commands for managing and using these tokens.
+
+### `kubeadm init` flags
+
+* `--token` If set, this injects the bootstrap token to use when initializing the cluster.  If this is unset, then a random token is created and shown to the user.  If set explicitly to the empty string then no token is generated or created.  This token is used for both discovery and TLS bootstrap.
```

Comment:
> This token is used for both discovery and TLS bootstrap.

suggest: This token has both the `usage-bootstrap-signing` and `usage-bootstrap-authentication` flags set to true, which means it will be used for both discovery and TLS bootstrap.